examples/go.mod

```diff
@@ -3,10 +3,10 @@ module google.golang.org/grpc/examples
 go 1.11
 
 require (
-	github.com/golang/protobuf v1.4.2
+	github.com/golang/protobuf v1.4.3
 	golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d
 	google.golang.org/genproto v0.0.0-20200806141610-86f49bd18e98
-	google.golang.org/grpc v1.31.0
+	google.golang.org/grpc v1.36.0
```

Comment:
Hmm, this is strange.

1. why does it change this file?
2. why is this updated to 1.36? Not the newer 1.37?

Reply:
This was probably related to the error message I pointed out earlier.
```
go: google.golang.org/grpc@v1.31.0 requires
        github.com/envoyproxy/go-control-plane@v0.9.9-0.20210512163311-63b5d3c536b0: missing go.sum entry; to add it:
        go mod download github.com/envoyproxy/go-control-plane
```

But I don't know why it has picked 1.36 and not 1.37.
I remember we either had some logic (or did it manually) to pin examples to use `latest-1` of the grpc version. Do you remember?

Reply:
We have this to pin to latest-1: https://github.com/grpc/grpc-go/blob/45e60095da54baad1e7ae28391941b64a40477e5/examples/go.mod#L13

Reply:
This was changed to 1.36 because of:
https://github.com/envoyproxy/go-control-plane/blob/cdae1e931d9270c35b5188e5f8e84db83b65c96d/go.mod#L15